### PR TITLE
Add useRootPaddingAwareAlignments to blank theme

### LINKED ISSUE #323: Add useRootPaddingAwareAlignments to the empty theme
For no-code users, there is no way to enable useRootPaddingAwareAlignments.
And I think it is common to want full-width blocks to be truly full-width, and still have spacing between the edge of the browser and the non-full-width content.

Could useRootPaddingAwareAlignments be enabled by default when "Create blank theme" is selected?

### PR DESCRIPTION
**Pull request by jffng**
Fixes #323, since there is no way to add this property via the global styles UI yet.

### CODE REVIEW

File: assets/boilerplate/theme.json [hunk 1]
```diff
@@ -1,40 +1,34 @@
...
+		"typography": {
+			"fontFamilies": [
+				{
+					"fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
+					"name": "System Font",
+					"slug": "system-font"
+				}
+			]
+		},
+		"useRootPaddingAwareAlignments": true
```
Comment (jffng) [inline assets/boilerplate/theme.json:21]: This is the only relevant change, the linter fixed the formatting of this file causing the rest of the changes.

**Review by vcanales — APPROVED** [review]
Works as expected, and the formatting is compliant. LGTM.

**Review by mikachan — APPROVED** [review]

**Comment by jffng** [comment]
Thanks @vcanales @mikachan !

**Comment by carolinan** [comment]
Thank you!

**Pull request merged**